x-pack/plugins/security_solution/common/utils/format_page_filter_search_param.test.ts

```diff
@@ -33,6 +34,7 @@ describe('formatPageFilterSearchParam', () => {
         fieldName: 'user.name',
         existsSelected: false,
         exclude: false,
+        hideActionBar: false,
```

Comment:
random question, but what is the action bar from the ui perspective?

Reply:
Here is an `action bar`of a filter control highlighted. By default it is hidden in `Status` and `Severity` controls.

<img width="1470" alt="Screenshot 2024-04-24 at 18 12 01" src="https://github.com/elastic/kibana/assets/7485038/ddd44208-8f9a-44b9-a2b4-69a1ef0b9303">